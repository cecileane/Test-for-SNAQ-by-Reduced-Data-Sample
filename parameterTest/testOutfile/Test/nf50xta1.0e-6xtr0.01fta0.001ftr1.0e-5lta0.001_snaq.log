optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.001,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 50, liktolAbs = 0.001.
rootname for files: nf50xta1.0e-6xtr0.01fta0.001ftr1.0e-5lta0.001_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 54211
seed: 54211 for run 1
Sat Oct  1 17:16:21 2016

 BEGIN SNaQ for run 1, seed 54211 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 147 (failures: 50) with -loglik=173.99254 and ht_min=[0.50506,9.84806,0.28297,0.3447,0.0,0.0,0.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 45		 45		 3	 |	 0		 0		 --
mvtarget	 43		 43		 1	 |	 1		 1		 --
chdir		 10		 10		 0	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 48		 5		 1	 |	 3		 3		 --
Total		 147		 104		 6	 |	 5		 5		 4
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.8

 FINISHED SNaQ for run 1, -loglik of best 173.99254072543775
(4,(5,#H7:0.0::0.49494397422943115):9.848064847979217,((1,2):0.3446958326169611,((3,6):0.0)#H7:0.0::0.5050560257705689):0.28296849650960276);
---------------------
seed: 29692 for run 2
Sat Oct  1 17:16:27 2016

 BEGIN SNaQ for run 2, seed 29692 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 69 (failures: 50) with -loglik=126.74251 and ht_min=[0.2856,10.0,1.51213,0.48942,0.51916]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 22		 22		 2	 |	 0		 0		 --
mvtarget	 31		 31		 1	 |	 0		 0		 --
chdir		 6		 6		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 9		 7		 1	 |	 1		 1		 --
Total		 69		 67		 6	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 126.74250981975037
(2,((3,(5,(6,#H7:::0.2856030291236419):10.0):0.5191607606002278):0.4894188664374484,4):1.5121337207213708,(1)#H7:::0.7143969708763581);
---------------------

Sat Oct  1 17:16:32 2016
MaxNet is (2,((3,(5,(6,#H7:::0.2856030291236419):10.0):0.5191607606002278):0.4894188664374484,4):1.5121337207213708,(1)#H7:::0.7143969708763581); 
with -loglik 126.74250981975037
