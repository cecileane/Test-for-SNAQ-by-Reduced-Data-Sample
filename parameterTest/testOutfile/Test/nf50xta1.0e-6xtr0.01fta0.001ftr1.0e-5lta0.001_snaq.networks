(2,((3,(5,(6,#H7:::0.2856030291236419):10.0):0.5191607606002278):0.4894188664374484,4):1.5121337207213708,(1)#H7:::0.7143969708763581);, with -loglik 126.74250981975037 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(2,((3,(5,(6)#H7:::0.6907343691938623):1.7275219336103047):0.4597563429985098,4):0.6650101310043699,(1,#H7:::0.3092656308061377):3.792961691981126);, with -loglik 128.26943113799743
(2,#H7:::0.4071421049506996,(1,(6,(5,(3,(4)#H7:::0.5928578950493004):0.017399328645075887):0.22632293539455195):1.145150114694927):1.7948876834536152);, with -loglik 173.89974468463373
(3,(5,(6,(1,(2)#H7:::0.5658617044613906):5.119447228833208):0.16666034920886155):0.029038995031479116,(#H7:::0.4341382955386094,4):1.7664267340700726);, with -loglik 191.9876810209484
(2,((3)#H7:::0.5978085326547721,4):0.06939043898131095,(1,(6,(5,#H7:::0.4021914673452279):9.989996817573491):0.9982723796865529):0.09512670202290127);, with -loglik 215.1238900316119
(2,((3,(5)#H7:::0.5104060694910459):1.0957373272134225,4):0.2063419551220964,(1,(6,#H7:::0.4895939305089541):5.394077236615038):0.04653200382175056);, with -loglik 231.27408259693706
