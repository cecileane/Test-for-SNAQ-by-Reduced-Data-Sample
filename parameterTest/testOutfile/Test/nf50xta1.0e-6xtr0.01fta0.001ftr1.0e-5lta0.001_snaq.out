(2,((3,(5,(6,#H7:::0.2856030291236419):10.0):0.5191607606002278):0.4894188664374484,4):1.5121337207213708,(1)#H7:::0.7143969708763581); -Ploglik = 126.74250981975037
 Dendroscope: (2,((3,(5,(6,#H7):10.0):0.5191607606002278):0.4894188664374484,4):1.5121337207213708,(1)#H7);
 Elapsed time: 11.691963934 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (2,((3,(5,(6,#H7:::0.2856030291236419):10.0):0.5191607606002278):0.4894188664374484,4):1.5121337207213708,(1)#H7:::0.7143969708763581);, with -loglik 126.74250981975037
 (4,(5,#H7:0.0::0.49494397422943115):9.848064847979217,((1,2):0.3446958326169611,((3,6):0.0)#H7:0.0::0.5050560257705689):0.28296849650960276);, with -loglik 173.99254072543775
-------
